src/dotnet-uninstall/Shared/BundleInfo/Versioning/MajorMinorSdkMinorVersion.cs

```diff
@@ -2,12 +2,17 @@
 
 namespace Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning
 {
-    public class MajorMinorSdkMinorVersion : MajorMinorVersion, IEquatable<MajorMinorSdkMinorVersion>, IComparable<MajorMinorSdkMinorVersion>
+    public class MajorMinorSdkMinorVersion : BeforePatch, IEquatable<MajorMinorSdkMinorVersion>, IComparable, IComparable<MajorMinorSdkMinorVersion>
```

Comment:
i was about to mention `MajorMinorSdkMinorVersion : MajorMinorVersion` is bad. nice change 👍